### DISCUSSION TITLE
请问可以自定义节点样式吗？

### DISCUSSION THREAD

**Discussion by yn22638** [hizzgdev/jsmind#543] [General]
您好，现在有个需求功能基本差不多，但是样式有些对不上，请问可以自定义节点样式吗，比如插槽什么的，写法是 vue2 的，因为需求比较紧，就看了一部分源码，功能是很充足的，绝对够用，很厉害
附上样式图
<img width="742" alt="image" src="https://github.com/hizzgdev/jsmind/assets/48940123/d31229e9-ef48-4166-b81d-2fc367919bd5">

**Comment by hizzgdev** [comment]
我看你这已经自定义样式了吧。可以先看看文档，文档里介绍了如何自定义主题和样式。

**Comment by hizzgdev** [comment]
文档里你可以关注这几个内容：

- [主题](https://github.com/hizzgdev/jsmind/blob/master/docs/zh/1.usage.md#13-%E4%B8%BB%E9%A2%98)
- [样式](https://github.com/hizzgdev/jsmind/blob/master/docs/zh/1.usage.md#14-%E6%A0%B7%E5%BC%8F)
- [view.custom_node_render](https://github.com/hizzgdev/jsmind/blob/master/docs/zh/2.options.md#23-%E6%8E%92%E7%89%88%E9%80%89%E9%A1%B9) : (function) 脑图节点的自定义渲染方法 (0.7.6 及以上版本支持) 

相信你一定有办法把样式调到这个样式的。另外还有两点，我感觉需要你自定义开发一下：
1. 节点后面的数字，这个现在是不支持的，你可能需要看看源码自己实现一下
2. 节点的连线，目前没有这样的线的样式，不过相对来说这块修改起来倒不复杂

**Comment by hizzgdev** [comment]
转到了 Discussion 区

**Comment by hizzgdev** [comment]
节点后面的数字已在 [v0.7.8](https://github.com/hizzgdev/jsmind/discussions/546) 版本中支持，欢迎试用，如有问题可随时反馈。感谢 🙏
